css-pseudo-classes/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Pseudo Classes</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+<h1>Hover Button</h1>
+<form>
+  <button type="button">I'm a Button!</button>
+</form>
```

Comment:
No `form` element is required here! We are not submitting any data, so there's no need for it. You can use the `button` element without a `form`